app/components/candidate_interface/offer_review_component.rb

```diff
@@ -32,6 +34,20 @@ def course_row
       }
     end
 
+    def fee_row
+      {
+        key: 'Fees',
+        value: fee_row_value,
+      }
+    end
+
+    def salary_row
+      {
+        key: 'Salary',
+        value: @course_choice.current_course.salary_details,
```

Comment:
Think this needs to rendered using a markdown rendered, [as on Find](https://github.com/DFE-Digital/find-teacher-training/blob/main/app/views/courses/_salary.html.erb#L4):

```suggestion
        value: markdown(@course_choice.current_course.salary_details),
```

Reply:
This is done too